### PR TITLE
Add information on SLOs and how to open a request

### PR DESCRIPTION
**Pull request by cblecker**
Add information on suggested SLOs and how to open a request with the GitHub Administration Team.

/area github-management
/cc @spiffxp @idvoretskyi @calebamiles @grodrigues3 @fejta

### CODE REVIEW

File: github-management/opening-a-request.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Bot/Automation issues
+
+If you need help with the following:
+- Bot configuration
+- Automatic merging
+- Label issues
+- Automation feedback and feature requests
+
+Please open an issue against the [kubernetes/test-infra] repository describing
+your issue. If your request is urgent, please escalate to the
```
Comment (fejta) [inline github-management/opening-a-request.md]: and/or contact `#testing-ops` on slack?

Comment (cblecker) [reply github-management/opening-a-request.md]: Done.

File: github-management/opening-a-request.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+[test-infra on-call] or reach out to #testing-ops on Slack.
```
Comment (fejta) [inline github-management/opening-a-request.md]: Can you backtick `#testing-ops` so it is more emphasized? 

https://github.com/kubernetes/community/blob/9d38dabf5dc53706731f3f0680e723271d86c964/github-management/opening-a-request.md

Comment (cblecker) [reply github-management/opening-a-request.md]: Done.

**Comment by spiffxp** [comment]
/hold cancel
We can iterate as needed

File: github-management/opening-a-request.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Please open an issue against the [kubernetes/org] repository describing your
+issue. If your request is urgent, please escalate to **[@kubernetes/owners]**.
+
+## Bot/Automation issues
+
+If you need help with the following:
+- Bot configuration
+- Automatic merging
+- Label issues
```
Comment (grodrigues3) [inline github-management/opening-a-request.md]: nit: Issue labeling

**Pull request merged**